### PR TITLE
BUG Ignore irrelevant warning from tensorflow v1.4

### PR DESCRIPTION
**Pull request by stephen-hoover**
This warning happens at import time. It doesn't indicate a problem and should be ignored.

### CODE REVIEW

**Comment by stephen-hoover** [comment]
Hm. The conda-forge package count failed. #51 is passing. This PR shouldn't have changed any packages, so I'll rebase once #51 merges.

**Comment by stephen-hoover** [comment]
@mheilman , this is passing tests and ready to review now.

File: Dockerfile [hunk 1]
```diff
@@ -105,7 +105,9 @@ RUN jupyter nbextension enable --py widgetsnbextension
 # https://github.com/joblib/joblib/blob/0.11/joblib/parallel.py#L328L342
 ENV JOBLIB_TEMP_FOLDER=/tmp
 
+# Filter out know irrelevant warnings.
```
Comment (mheilman) [inline Dockerfile]: could you please add a TODO comment here saying that this can be removed when TF is upgraded?

Comment (stephen-hoover) [reply Dockerfile]: Done.

**Comment by stephen-hoover** [comment]
I'm experimenting more with this. It looks like it will suppress warnings which come from interactive sessions, but not warnings when you run things as scripts, e.g. `python myscript.py`. That makes this much less useful.

**Comment by stephen-hoover** [comment]
Yeah, this won't work. https://docs.python.org/3.6/tutorial/appendix.html#the-interactive-startup-file

**Comment by stephen-hoover** [comment]
@mheilman , I switched from the startup file to `usercustomize.py`, https://docs.python.org/3.6/tutorial/appendix.html#the-customization-modules . This should do the same thing I was trying to do, but in all invocations of python, not just interactive sessions.

**Review by mheilman — APPROVED** [review]
I didn't know about `usercustomize.py`. nice find.  LGTM

**Pull request merged**